src/argilla_sdk/records/_resource.py

```diff
@@ -253,7 +252,7 @@ def from_dict(
 
         suggestions = [Suggestion(**suggestion_dict) for suggestion_dict in suggestion_values.values()]
 
-        return cls(
+        return Record(
```

Comment:
I think is fine to keep the `cls`. If someday the `Record` class is extended, this method may not work as expected.